### ISSUE TITLE
custom view for progress  in Stories Progress View   tabs 

### ISSUE THREAD

**Issue by hemantmoudgil** [shts/StoriesProgressView#43]
I like   the library  though but one issue  can I customize the  progress  tabs view in StoriesProgressView  because want to use a custom view in progress tabs

**Comment by ZayniddinMamarasulov** [comment]
Yes, I also want to change progress view color dark when showing picture with white background. Because progress view is not visible with white pictures
